mocktarget/errors.json

```diff
@@ -120,6 +120,26 @@
                     }
                 }
             ]
+        },
+        {
+            "aggregation_key": "info-message",
+            "total_count": 1,
+            "severity": "info",
+            "created_at": "2019-09-15T11:46:25.333Z",
+            "latest_errors": [
```

Comment:
for info messages "error" is not the best name (maybe events would be better), but changing it will imply a breaking API change and will need to be considered carefully